packages/mdl-button/mdl-button.scss

```diff
@@ -0,0 +1,152 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import "variables";
+
+.mdl-button {
+  display: inline-block;
+  position: relative;
+  min-width: 64px;
+  height: 36px;
+  padding: 0 16px;
+  border: none;
+  border-radius: 2px;
+  outline: none;
+  background: none;
+  color: rgba(0, 0, 0, .87);
+  font-family: Roboto, sans-serif;
+  font-size: .875rem; /* 14sp */
```

Comment:
Shouldn't the font be defined in `dp` (`px`), since the rest of the button doesn't scale with text size? It's ambiguous in the spec, but I think that it's likely what they meant, since it wouldn't make sense to have text overflow if the user doubles the default font size, for example.